### PR TITLE
added method to specify pages with title and revid

### PR DESCRIPTION
**Pull request by ayuhsya**
Can instantiate WikiPage class with parameters like:

`wk = WikiPage(titles="pantera")`

or

`wk = WikiPage(pageids="142805")`

Also works with revids. Querying with Generators has not been added.

### CODE REVIEW

**Comment by abinashmeher999** [comment]
In your current implementation, the user would have to pass in a `|` separated list of pages. It would be better if the user could pass in the python array as an input. Also the titles have to be [percent encoded](https://www.mediawiki.org/wiki/API:Main_page#Action-specific_parameters) before they are sent via the request.

**Comment by ayuhsya** [comment]
Okay, I'll do the array thing but percent encoding is not necessary. I've tried calling API without percent encoding and it works. It is self normalizing.

File: sample/core.py [hunk 1]
```diff
@@ -101,8 +104,14 @@ def _append_results(currlist, newlist, prop, strip_chars):
...
+    except KeyError as error:
+        print (error.args)
+        sys.exit("Exited!")
 
-    wk = WikiPage('843158','20715044')
     pprint(wk.get_categories())
-    pprint(wk.get_images())
-    pprint(wk.get_linkshere())
+    #pprint(wk.get_images())
+    #pprint(wk.get_linkshere())
```
Comment (abinashmeher999) [inline sample/core.py]: Leave them uncommented and this Pull Request would be good to go.

File: sample/core.py [hunk 2]
```diff
@@ -80,6 +73,16 @@ def get_linkshere(self, lhprop="pageid|title|redirect", lhlimit="max"):
...
 
+    def _parse_kwargs(self, **kwparams):
+        if 'pageids' in kwparams:
+            self.payload['pageids'] = '|'.join(str(ID) for ID in list(kwparams['pageids']))
+        elif 'titles' in kwparams:
+            self.payload['titles'] = '|'.join(str(ID) for ID in list(kwparams['titles']))
+        elif 'revids' in kwparams:
+            self.payload['revids'] = '|'.join(str(ID) for ID in list(kwparams['revids']))
+        else:
+            raise KeyError("No valid arguments passed!")
```
Comment (abinashmeher999) [inline sample/core.py]: Sorry I forgot to mention this the first time. This one should be a [`ValueError`](https://docs.python.org/2/library/exceptions.html#exceptions.ValueError) which is preferred when the argument is invalid.

**Review by abinashmeher999 — CHANGES_REQUESTED** [review]

File: sample/core.py (hunk 2 above)
Comment (ayuhsya) [reply sample/core.py]: @abinashmeher999 No problem, I'm getting more and more commits. :smiling_imp:

Comment (abinashmeher999) [reply sample/core.py]: @ayuhsya You forget I can squash and merge too :smiling_imp:

**Review by abinashmeher999 — APPROVED** [review]

File: sample/core.py (hunk 2 above)
Comment (ayuhsya) [reply sample/core.py]: @abinashmeher999 Plis -^-

**Pull request merged**